Common/HKUnit.swift

```diff
@@ -6,25 +6,17 @@
 //  Copyright © 2016 Nathan Racklyeft. All rights reserved.
 //
 
-import HealthKit
-
-extension HKUnit {
-    static let milligramsPerDeciliter: HKUnit = {
-        HKUnit.gramUnit(with: .milli).unitDivided(by: .literUnit(with: .deci))
-    }()
-
-    static let millimolesPerLiter: HKUnit = {
-        HKUnit.moleUnit(with: .milli, molarMass: HKUnitMolarMassBloodGlucose).unitDivided(by: .liter())
-    }()
+import LoopAlgorithm
 
+extension LoopUnit {
     var localizedShortUnitString: String {
-        if self == HKUnit.millimolesPerLiter {
+        if self == LoopUnit.millimolesPerLiter {
             return NSLocalizedString("mmol/L", comment: "The short unit display string for millimoles of glucose per liter")
         } else if self == .milligramsPerDeciliter {
             return NSLocalizedString("mg/dL", comment: "The short unit display string for milligrams of glucose per decilter")
-        } else if self == .internationalUnit() {
+        } else if self == .internationalUnit {
             return NSLocalizedString("U", comment: "The short unit display string for international units of insulin")
-        } else if self == .gram() {
+        } else if self == .gram {
```

Comment:
[nit] is this needed? Or can this be defined in LoopAlgorithm?

Reply:
Moved :)